packages/@o3r/analytics/schematics/analytics-to-component/schema.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "NgAddAnalyticsSchematicsSchema",
+  "title": "Add Otter Analytics to an existing component",
+  "description": "Add Otter Analytics to an existing component",
+  "properties": {
+    "path": {
+      "type": "string",
+      "description": "Path to the component"
+    },
+    "skipLinter": {
+      "type": "boolean",
+      "description": "Skip the linter process",
+      "default": false
+    },
+    "activateDummy": {
+      "type": "boolean",
+      "description": "Generate dummy values",
+      "default": false,
+      "x-prompt": "Generate dummy analytics event"
```

Comment:
do we want to keep this prompt?

Reply:
I keep this prompt for others schematics 
maybe I can remove it in another PR, to keep it all aligned
(I will do another PR to refactor a bit all the component schematics code to share a maximum of lines, that we didn't think we can share at the beginning)

Reply:
Ok then, let's remove it in another PR